pkg/cvo/cvo.go

```diff
@@ -160,6 +163,16 @@ func New(
 
 		queue:                 workqueue.NewNamedRateLimitingQueue(workqueue.DefaultControllerRateLimiter(), "clusterversion"),
 		availableUpdatesQueue: workqueue.NewNamedRateLimitingQueue(workqueue.DefaultControllerRateLimiter(), "availableupdates"),
+		proxyExists:           false,
+	}
+
+	_, err := client.ConfigV1().Proxies().Get("cluster", metav1.GetOptions{})
+
+	if err == nil {
+		optr.proxyLister = proxyInformer.Lister()
+		proxyInformer.Informer().AddEventHandler(optr.eventHandler())
+		optr.cacheSynced = append(optr.cacheSynced, proxyInformer.Informer().HasSynced)
```

Comment:
just drop the proxy from cachedSynced... it's goal is to make sure we don't start action on inconsistent cache... but there's no such requirement for proxy informer.